src/features/donations/components/Donation/ShareDonation/ShareDonation.js

```diff
@@ -0,0 +1,72 @@
+import {

```

Comment:
Declare this component into the `components` folder

Reply:
Ok